recipes/deap/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0.2.post2" %}
+{% set gh_org = "DEAP" %}
+{% set gh_repo = "deap" %}
+{% set pkg_name = "DEAP" %}
+{% set commit = "a4dc75208662ce40158ba0c9eb0045d0bcc66d70" %}
```

Comment:
I am not a big fan of all that jinja-ninja stuff :smile: (But that won't prevent me from merging this.)